File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 Modul 6/C/PRAK604-2410817110006-HamkaArifani.c
#include <stdio.h>
#include <string.h>
int main(){
    char kode[999], pesan[999];
    int panjangkode,panjangpesan, panjangbintang, panjangpagar;
    panjangbintang=0;
    panjangpagar=0;
    scanf(" %[^\n]",&kode);
    scanf(" %[^\n]",&pesan);
    panjangkode=strlen(kode);
    panjangpesan=strlen(pesan);
    if(panjangkode!=panjangpesan){
        printf("Panjang kalimat berbeda, pesan palsu");
    } else{
        for(int h=0;h<panjangpesan;h++){
            if(kode[h]==' ' && pesan[h]==' '){
                continue;
            }
            if(kode[h]==pesan[h]){
                printf("*");
                panjangbintang++;
            }
            else {
                printf("#");
                panjangpagar++;
            }
        }
        printf("\n");
        printf("*= %d\n",panjangbintang);
        printf("#= %d\n",panjangpagar);
        if(panjangbintang>=panjangpagar){
            printf("Pesan asli");
        }
        else{
            printf("Pesan palsu");
        }
    }
}

File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 Modul 6/Python/PRAK604-2410817110006-HamkaArifani.py
kode=input("")
pesan=input("")
panjangkode=len(kode)
panjangpesan=len(pesan)
panjangbintang=0
panjangpagar=0
if panjangkode != panjangpesan:
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for h in range (panjangkode):
        if pesan[h]==" " and kode[h]==" ":
            continue
        if kode[h]==pesan[h]:
            print("*", end="")
            panjangbintang+=1
        else:
            print("#",end="")
            panjangpagar+=1
    print(f"\n*= {panjangbintang:.0f}")
    print(f"*= {panjangpagar:.0f}")
    if panjangbintang>=panjangpagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")